components/rhel/rhel-ose/.cdk

```diff
@@ -0,0 +1,4 @@
+openshift.auth.scheme=Basic
+openshift.auth.username=openshift-dev
+openshift.auth.password=devel
+cdk.box.version=2.3
```

Comment:
@praveenkumar Why the box version is 2.3 here? Does it represent the upcoming version?